Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -299,6 +299,60 @@ def get_impacted_resources(self, cve: str, resource_type: str) -> dict:
         )
 
 
+    def get_waas_policies(self) -> dict:
+        """
+        Get the current WAAS policy
+
+        Returns:
+            dict: the current policy.
+        """
+        return self._http_request(
+            method="GET", url_suffix="policies/firewall/app/container"
+        )
+
+
+    def update_waas_policies(self, policy: dict) -> dict:
+        """
+        Update the waas policy.
+
+        Args:
+            policy (dict): the previous waas policy.
+
+        Returns:
+            dict: the updated policy.
+        """
+        return self._http_request(
+            method="PUT", url_suffix="policies/firewall/app/container", json_data=policy, resp_type="response"
+        )
+
+
+    def get_firewall_audit_container_alerts(self, image_name: str, from_time: str, to_time: str, limit: int, audit_type: str):
+        """
+        Get the container audit alerts for a specific image.
+
+        Args:
+            image_name (str): The container image name.
+            from_time (str): The start time of the query for alerts.
+            to_time (str): The end time to query alerts.
+            limit (num): the limit of alerts returned.
+            audit_type (str): the alert audit type.
+
+        Returns:
+            dict: the container alerts.
+        """
+        params = {
+            "type": audit_type,
+            "imageName": image_name,
+            "from": from_time,
+            "to": to_time,
+            "limit": limit
+        }
+        return self._http_request(
+            method="GET", url_suffix="audits/firewall/app/container", params=params, resp_type="response"
+        ).json()
```

Comment:
```suggestion
        return self._http_request(
            method="GET", url_suffix="audits/firewall/app/container", params=params
        )
```
will return `json` by default as the `resp_type` default value is json.